tests/bsim/bluetooth/host/misc/sample_test/src/main.c

```diff
@@ -9,11 +9,33 @@
 #include "bs_tracing.h"
 #include "bstests.h"
 #include "babblekit/testcase.h"
+#include "testlib/log_utils.h"
+
 
 extern void entrypoint_dut(void);
 extern void entrypoint_peer(void);
 extern enum bst_result_t bst_result;
 
+static uint8_t	set_log_level_run_time = 3;
```

Comment:
This doesn't need to be a global variable

Reply:
it was my understanding from the description. i can make it local to the `test_args` function